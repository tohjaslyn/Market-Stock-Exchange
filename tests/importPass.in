add trader1 100
import trader1 DEF 10
exit
