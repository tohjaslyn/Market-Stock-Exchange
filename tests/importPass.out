$ Success.
$ Trader now has 10.00 units of DEF.
$ Have a nice day.
